### ISSUE TITLE
Build failed on MacOS Big Sur

### ISSUE THREAD

**Issue by douglarek** [macvim-dev/macvim#1113]
```
$ ./configure --with-features=huge --enable-multibyte --with-tlib=ncurses --enable-cscope --enable-terminal --with-compiledby=Steps --enable-python3interp --enable-luainterp --with-lua-prefix=/usr/local/ --enable-fail-if-missing --prefix=/usr/local --with-macarchs=x86_64 --disable-sparkle
```

make error:

```
error: The linked and embedded framework 'Sparkle.framework' is missing one or more architectures required by this target: arm64. (in target 'MacVim' from project 'MacVim')
```

OS:

```
MacOS Big Sur 11.0.1 Release
Xcode 12.2 release
Python 3.8.1
```

**Comment by limintang** [comment]
An informal solution:

Open src/MacVim/MacVim.xcodeproj with Xcode, in Build Settings -> Architectures -> Excluded Architectures -> MacVim, add `arm64`.

**Comment by douglarek** [comment]
@limintang thanks, I am not familiar with MacOS development, would you like to share this diff ?

**Comment by limintang** [comment]
While you could modify MacVim.xcodeproj in text editor, modify it in Xcode is more straightforward:

<img width="1419" alt="Screen Shot" src="https://user-images.githubusercontent.com/9778963/99162653-f0272c80-26b4-11eb-8a61-033dde831ad4.png">

**Comment by douglarek** [comment]
@limintang thanks, I have got this diff:

```diff
diff --git a/src/MacVim/MacVim.xcodeproj/project.pbxproj b/src/MacVim/MacVim.xcodeproj/project.pbxproj
index 729c23009..9b66f5335 100644
--- a/src/MacVim/MacVim.xcodeproj/project.pbxproj
+++ b/src/MacVim/MacVim.xcodeproj/project.pbxproj
@@ -3,7 +3,7 @@
        archiveVersion = 1;
        classes = {
        };
-       objectVersion = 47;
+       objectVersion = 54;
        objects = {

 /* Begin PBXBuildFile section */
@@ -1169,6 +1169,7 @@
                        buildSettings = {
                                COMBINE_HIDPI_IMAGES = YES;
                                COPY_PHASE_STRIP = NO;
+                               EXCLUDED_ARCHS = arm64;
                                FRAMEWORK_SEARCH_PATHS = (
                                        "$(inherited)",
                                        "$(FRAMEWORK_SEARCH_PATHS_QUOTED_FOR_TARGET_1)",
@@ -1202,6 +1203,7 @@
                        buildSettings = {
                                COMBINE_HIDPI_IMAGES = YES;
                                COPY_PHASE_STRIP = YES;
+                               EXCLUDED_ARCHS = arm64;
                                FRAMEWORK_SEARCH_PATHS = (
                                        "$(inherited)",
                                        "$(FRAMEWORK_SEARCH_PATHS_QUOTED_FOR_TARGET_1)",
@@ -1233,6 +1235,7 @@
                        buildSettings = {
                                CLANG_ANALYZER_LOCALIZABILITY_NONLOCALIZED = YES;
                                ENABLE_TESTABILITY = YES;
+                               EXCLUDED_ARCHS = arm64;
                                GCC_VERSION = 4.2;
                                GCC_WARN_ABOUT_RETURN_TYPE = YES;
                                GCC_WARN_UNUSED_VARIABLE = YES;
@@ -1246,6 +1249,7 @@
                        isa = XCBuildConfiguration;
                        buildSettings = {
                                CLANG_ANALYZER_LOCALIZABILITY_NONLOCALIZED = YES;
+                               EXCLUDED_ARCHS = arm64;
                                GCC_VERSION = 4.2;
                                GCC_WARN_ABOUT_RETURN_TYPE = YES;
                                GCC_WARN_UNUSED_VARIABLE = YES;
```

**Comment by ychin** [comment]
I think this is fixed by #1118 so ARM builds will work now.

**Comment by ychin** [comment]
That said, if you are building locally and want to build faster or have a leaner version of the app it's probably still desirable to exclude ARM though.
